src/rsa/rsa.rs

```diff
@@ -26,6 +26,7 @@ use std;
 
 use untrusted;
 
+pub const RSA_KEY_MAX_BITS: usize = 4096;
```

Comment:
I think this should be named so that it is clear it refers to the public key modulus, e.g. `RSA_PUBLIC_KEY_MODULUS_BITS_MAX`.